src/main/java/com/atherys/quests/command/quest/AttachQuestToItemCommand.java

```diff
@@ -1,4 +1,61 @@
 package com.atherys.quests.command.quest;
 
-public class AttachQuestToItemCommand {
+import com.atherys.core.command.ParameterizedCommand;
+import com.atherys.core.command.annotation.Aliases;
+import com.atherys.core.command.annotation.Description;
+import com.atherys.quests.api.quest.Quest;
+import com.atherys.quests.data.QuestData;
+import com.atherys.quests.service.QuestService;
+import com.atherys.quests.util.QuestMsg;
+import org.spongepowered.api.command.CommandException;
+import org.spongepowered.api.command.CommandResult;
+import org.spongepowered.api.command.CommandSource;
+import org.spongepowered.api.command.args.CommandContext;
+import org.spongepowered.api.command.args.CommandElement;
+import org.spongepowered.api.command.args.GenericArguments;
+import org.spongepowered.api.data.type.HandTypes;
+import org.spongepowered.api.entity.living.player.Player;
+import org.spongepowered.api.item.inventory.ItemStack;
+import org.spongepowered.api.text.Text;
+
+import javax.annotation.Nonnull;
+import java.util.Optional;
+
+@Aliases("toitem")
+@Description("Sets a quest to the item in hand.")
+public class AttachQuestToItemCommand implements ParameterizedCommand {
```

Comment:
Add an @Permission annotation, this seems like a command which should be restricted from access to regular players.